weldx/measurement.py

```diff
@@ -98,7 +97,7 @@ def _add_node(
         if parent_node is not None:
             graph.add_edge(parent_node, node)
 
-    def plot(self, axes: "Axes" = None):  # pragma: no cover
```

Comment:
Why is it removed? We can simply fix the doc issue by adding it to the `nitpick_ignore`

Reply:
it didn't work for me in sphinx but it works in other places (geometry.py)

I think it is pretty self-explanatory as is so we can fix that in a later PR if necessary